core/src/main/scala/org/apache/spark/storage/MemoryStore.scala

```diff
@@ -371,7 +371,7 @@ private[spark] class MemoryStore(blockManager: BlockManager, maxMemory: Long)
   private def ensureFreeSpace(
       blockIdToAdd: BlockId,
       space: Long): ResultWithDroppedBlocks = {
-    logInfo(s"ensureFreeSpace($space) called with curMem=$currentMemory, maxMem=$maxMemory")
+    logDebug(s"ensureFreeSpace($space) called with curMem=$currentMemory, maxMem=$maxMemory")
```

Comment:
This one I'd say is fairly important. I've used it somewhat often in production clusters to understand when things are dropping to disk and why.